docs/cart-and-checkout-blocks/removing-checkout-fields.md

```diff
@@ -0,0 +1,197 @@
+---
+post_title: Cart and Checkout - Removing checkout fields
+menu_title: Extensibility
+tags: reference
+---
+<!-- markdownlint-disable MD041 -->
+
+If you're trying to extend the new Checkout block, you might notice the previous `checkout_fields` isn't working. This is intentional, we're taking back control of how to render and handle fields, yet, one of the most common requests we get is how to disable Checkout fields for certain order types. For a long time, our answer was "you can't". You actually can, but we don't encourage it.
```

Comment:
Not sure if we want to include this verbatim? I'm more comfortable speaking like this in my blog but dunno from Woo? Let's see keep it and see what dev advocacy say later.

Reply:
I reworded it.